### PR TITLE
Favicon Update

### PR DESCRIPTION
**Pull request by JasonHoku**

### CODE REVIEW

**Comment by bkesecker** [comment]
I'll get this loaded in test soon.

File: client/public/index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.gif" />
```
Comment (InoUno) [inline client/public/index.html:5]: This should still be `.ico`? The file that was added is an `.ico`.

Comment (JasonHoku) [reply client/public/index.html:5]: I think I missed a 'git add .' during the final revision of that PR work.

Thanks for the careful eye, Ino. Fixed in this PR.

Not all browsers will support .gif favicons yet without strenuous javascripting so I've included reduceds of both for HTML5 browsers to choose their preference with the shortcut icon tag pointing to the gif.

Comment (InoUno) [reply client/public/index.html:5]: I don't think animated favicons are good to have, so I'd prefer to just have the `.ico`.

**Review by bkesecker — APPROVED** [review]
Working correctly on test server.

**Pull request merged**